src/observer.ts

```diff
@@ -1,5 +1,36 @@
 import { Subscription } from './subscription';
 
+/** Symbol.observable addition */
+/* Note: This will add Symbol.observable globally for all TypeScript users,
+  however, we are no longer polyfilling Symbol.observable */
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol;
+  }
+}
+
+/** Symbol.observable or a string "@@observable". Used for interop */
+//tslint:disable-next-line
+const { observable: Symbol_observable } = require('rxjs/symbol/observable');
+
+let symbolObservable = Symbol_observable;
+if (typeof Symbol !== 'undefined') {
+  // Older versions of Rx will polyfill Symbol.observable, which gets
+  // compiled into our UMD bundle. At runtime, our UMD bundle defines its
+  // version of Symbol.observable, and since it's not getting it from Rx via
+  // `require()` anymore, Rx defines and looks for a different Symbol.observable,
+  // instance, leading to mismatches.
+  // Assigning the global Symbol.observable to the one we bundle in here means
+  // that Rx's polyfill will pick it up. Alternatively if there's already a global
+  // Symbol.observable (like if Rx was required first), we should use that one inside Ix.
+  if (typeof Symbol['observable'] === 'undefined') {
+    (Symbol as any)['observable'] = symbolObservable;
```

Comment:
Imo a library should never polyfill globals. This should be provided by the user, for the exact reason described above: to not conflict with other libraries. Iirc Rx stopped polyfilling it for this reason too.

Reply:
I don't disagree. And to clarify, _we_ aren't polyfilling Symbol.observable here (rxjs@5.5.11 is), we're just switching to use the global if someone else is. Maybe the real solution is to update to rxjs@6.x.

Reply:
We should definitely update in any case